2020/10/2020-10-23-RIAA.md

```diff
@@ -4,115 +4,27 @@ GitHub
 
 Dear Sir or Madam:
 
-I am contacting you on behalf of the Recording Industry Association of America, Inc. (RIAA) and
-its member record companies. The RIAA is a trade association whose member companies
-create, manufacture or distribute sound recordings representing approximately eighty-five (85)
-percent of all legitimate recorded music consumption in the United States. Under penalty of
-perjury, we submit that the RIAA is authorized to act on behalf of its member companies on
-matters involving the infringement of their sound recordings, audiovisual works and images,
-including enforcing their copyrights and common law rights on the Internet.
+We here at the RIIA are complete and utter idiots, and we have no idea how anything works.
 
-Copyright Violations. We have learned that your service is hosting the youtube-dl source code
-on its network at the following locations, among others:
+As we have a monopoly on 85% of the music market in America, we've basically got you fucks by the balls, and you can't do jack shit about it, because we have our artists by the balls too.
 
-https://github.com/ytdl-org/youtube-dl  
-https://github.com/ytdl-org/youtube-dl/tree/gh-pages  
-http://ytdl-org.github.io/youtube-dl/  
-https://github.com/benkeung/youtube-dl  
-https://github.com/cyberjacob/youtube-dl  
-https://github.com/elaopinska/youtube-dl  
-[private]    
-https://github.com/huangciyin/youtube-dl  
-https://github.com/jckelley/youtube-dl  
-https://github.com/LouisPlisso/youtube-dl  
-https://github.com/ojauch/youtube-dl  
-https://github.com/rbrito/youtube-dl  
-https://github.com/successLee/youtube-dl  
-https://github.com/trammel/youtube-dl  
-https://github.com/vs9390/youtube-dl  
-https://github.com/zackfern/youtube-dl  
-https://github.com/tosuch/youtube-dl  
-https://github.com/pornophage/youtube-dl  
-https://github.com/tejaskhot/youtube-dl  
-https://github.com/VideoUtils/youtube-dl  
-https://github.com/798221028/youtube-dl  
+Since we all know American law applies to the entire world, we can do whatever the fuck we want lol
```

Comment:
They do-ish. A lot of internet services are hosted in USA and so under american law... which sucks but i agree that it feels like an abuse of the law to take down lawful code and software

Reply:
So what? If they will sue and the guy (imagine in Russia) will show up in court, it will lack jurisdiction and dismissed with prejudice.

Reply:
@ValZapod it's cute that you think that. hey remember all the times the cia assassinated foreign leaders so the us could create puppet states?

the us doesn't care about jurisdiction, and will enforce its laws as it pleases